README.md

```diff
@@ -145,37 +143,16 @@ If everything above worked, LetsLapse is installed. In a browser on your network
 ## What have we done? What's LetsLapse
 
 
-
 ## Additional Notes 
 For, um, cleaning up... 
 
-### Possible depencies 
+### Dependencies installation 
 ```
-sudo apt install python3-pip -y
-sudo apt install python3-pip
+sudo apt-get update && apt install python3-pip -y
+sudo apt install libopenjp2-7 libopenjp2-7-dev libopenjp2-tools libatlas-base-dev
 python3 -m pip install --upgrade pip
-python3 -m pip install --upgrade Pillow
-python3 -m pip install --upgrade Pillow --global-option="build_ext" --global-option="--enable-[feature]"
-sudo apt install libopenjp2-7 libopenjp2-7-dev libopenjp2-tools
-```
-
-#### Additional requirements to run blending tools
+python3 -m pip install -r requirements.txt
 ```
-sudo pip3 install opencv
-sudo apt-get install libatlas-base-dev
-sudo pip3 install piexif
-```
-
-#### For opencv and image processing
-
-```
-python3 -m pip install --upgrade imutils
-python3 -m pip3 install opencv-contrib-python
-python3 -m pip install opencv-python - this worked, not contrib version
```

Comment:
You mention two here

> python3 -m pip3 install opencv-contrib-python
> python3 -m pip install opencv-python - this worked, not contrib version

better to edit the `requirements.txt` with the right/prefered one

Reply:
Thanks, yep - I'll look into requirements.txt - I didn't know about it, but knew my dependency integration wasn't right.

Equally, seems that issues I'm having with performance are related to the install method (i.e. pip3 vs apt-get). This is an area I'm unsure about and testing.

Reply:
ok - I'm just keeping my head above water with git - excuse my poor form. I'm merging this with the dev branch for now, I think. Cheers Leigh!